### PR TITLE
IS-13789: document the added 'entity' and 'source_entity' bound param…

### PR DESCRIPTION
**Pull request by tombech**
…eters to jinja templates

### CODE REVIEW

File: hub/changelog.rst [hunk 1]
```diff
@@ -1,14 +1,21 @@
 Changelog
 =========
 
+
+.. _changelog_2023_01_18:
+
+2023-01-18
+----------
+* Added "entity" and "source_entity" as bound parameters various Jinja templateable properties in the :ref:`REST system <rest_system>`, :ref:`REST transform <rest_transform>`, :ref:`REST source <rest_source>` and :ref:`REST sink <rest_sink>`.
```
Comment (branislavjenco) [inline hub/changelog.rst:9]: Is there some word missing? ..."as bound parameters various Jinja"..

File: hub/documentation/service-configuration/pipes/configuration-sources-rest.rst [hunk 2]
```diff
@@ -142,11 +142,10 @@ Properties
        if missing from the source system. Note that this property can be defined
        in the specified ``operation`` section of the :ref:`REST system <rest_system>` as well. The source configuration
        will take precedence if defined. The bound parameters available to this template are ``body``, ``url``,
-       ``requests_params``, ``properties``, ``since`` and ``headers``. All entity properties are also available as named
-       variables. If the operation supports paging then ``previous_body`` and ``previous_headers`` are available for all
-       page requests except the first. Tip: use Jinja's
-       `"is defined" <https://jinja.palletsprojects.com/en/3.1.x/templates/#tests>`_ tests for these variables to set
-       default values for the first page.
+       ``requests_params``, ``properties``, ``since``, ``entity`` and ``headers``. If the operation supports paging
+       then ``previous_body`` and ``previous_headers`` are available for allvpage requests except the first. Tip: use
```
Comment (knutj42) [inline hub/documentation/service-configuration/pipes/configuration-sources-rest.rst:146]: typo: allvpage

File: hub/changelog.rst (hunk 1 above)
Comment (knutj42) [inline hub/changelog.rst:9]: typo: missing "in" between "parameters" and  "various"

File: hub/documentation/service-configuration/pipes/configuration-transforms-rest.rst [hunk 3]
```diff
@@ -163,10 +163,10 @@ Properties
        the ``payload`` exists in the entity, then the one in the entity will take
        precedence over any ``payload`` defined on transform or system operation (in that order). This property
        supports the ``Jinja`` template (https://palletsprojects.com/p/jinja/) syntax with the named variables
-       ``properties``, ``url``, ``request_params`` and ``headers`` available to the template. If the operation supports
-       paging then ``previous_body`` and ``previous_headers`` are available for all page requests except the first.
-       Tip: use Jinja's `"is defined" <https://jinja.palletsprojects.com/en/3.1.x/templates/#tests>`_ tests for these
-       variables to set default values for the first page.
+       ``properties``, ``url``, ``request_params``, ``entity``, ``source_entity`` and ``headers`` available to the template.
```
Comment (knutj42) [inline hub/documentation/service-configuration/pipes/configuration-transforms-rest.rst:166]: Does the `payload` template actually have a `source_entity` variable? As far as I can tell from the code it only has a `entity` variable (which is the source entity).

**Review by branislavjenco — APPROVED** [review]

File: hub/documentation/service-configuration/pipes/configuration-transforms-rest.rst (hunk 3 above)
Comment (tombech) [reply hub/documentation/service-configuration/pipes/configuration-transforms-rest.rst]: You're right, it doesn't

**Pull request merged**